pkg/tests/integration/test_salt_user.py

```diff
@@ -61,14 +108,54 @@ def test_salt_cloud_dirs(install_salt):
     """
     Test the correct user is running the Salt Master
     """
+    if platform.is_windows() or platform.is_darwin():
+        pytest.skip("Package does not have user set. Not testing user")
     paths = [
-        "/opt/saltstack/salt/lib/python{}.{}/site-packages/salt/cloud/deploy".format(
-            *sys.version_info
-        ),
+        "/opt/saltstack/salt/lib/python3.10/site-packages/salt/cloud/deploy",
```

Comment:
```suggestion
        "/opt/saltstack/salt/lib/python3.10/site-packages/salt/cloud/deploy",
```
Don't hardcode the python version. Why did you need to change this line?

Reply:
Just accidentally stomped on that - reverted.

Reply:
I made the change in https://github.com/saltstack/salt/pull/64194/commits/1b411c7fb5cb4d07ddb24dae8b6c18953ea4d987 a couple of days before your comment.

Reply:
> I made the change in [1b411c7](https://github.com/saltstack/salt/commit/1b411c7fb5cb4d07ddb24dae8b6c18953ea4d987) a couple of days before your comment.

There's one more hardcoded reference, the one being commented on

Reply:
@s0undt3ch - apologies, you're correct. I've actually removed test_salt_cloud_dirs() as that is now covered by  test_pkg_paths()